handler/handler.go

```diff
@@ -50,7 +98,11 @@ func (q *NatsQueue) Queue(req *queue.Request) error {
 		log.Println(err)
 	}
 
-	err = q.nc.Publish(q.Topic, out)
+	q.ncMutex.RLock()
+	nc := q.nc
+	q.ncMutex.RUnlock()
+
+	err = nc.Publish(q.Topic, out)
```

Comment:
Some linters will probably ask you to do a `return nc.Publish(q.Topic, out)` here since there is no value in storing in `err` and then simply returning `err`.

Reply:
Changed